StyleCop.Analyzers/StyleCop.Analyzers.CodeFixes/DocumentationRules/FileHeaderCodeFixProvider.cs

```diff
@@ -139,8 +140,7 @@ private static SyntaxNode ReplaceWellFormedMultiLineCommentHeader(Document docum
             triviaStringParts[0] = commentIndentation + interlinePadding + " " + triviaStringParts[0];
             StringBuilder sb = StringBuilderPool.Allocate();
             string fileName = Path.GetFileName(document.FilePath);
-            var copyrightText = commentIndentation + interlinePadding + " " +
-                GetCopyrightText(commentIndentation + interlinePadding, settings.DocumentationRules.GetCopyrightText(fileName), newLineText);
```

Comment:
:memo: This was the only change in the second revision (I amended the original commit to fix build test errors)